index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  </head>
+  <body>
+
+   <div class="app-wrap">
+     <header class="app-header">
+         <h1>Welcome</h1>
+         <figure>
+           <img src="images/Union.png" alt="union image">
+         </figure>
```

Comment:
- [ ] Great job making this section a `<header>`. But to keep your HTML semantic I would suggest the following structure:

```
<header class="app-header">
  <a>Welcome</a>
  <nav><i class="fas fa-bars"></i></nav>
</header>
```
- [ ] I wouldn't suggest the use of a `<h1>` in this case, mainly, for SEO purposes, you should only have one h1 tag. With that in mind, the h1 should concisely describe the overall purpose of the content, so there will be a better use for along the page! You can read more about heading [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
- [ ] Considering you added `font-awesome` to your project, you should consider adding the bars icon here. Icons are more personalizable (you can easily change colors and size), so when possible, should be preferred over images!
- [ ] Even though the menu was not implemented yet, to keep the HTML semantic, you should consider having it wrapped in a `<nav>` tag. Once you implement the JavaScript, this will help the user navigate through your website, so makes sense to keep it semantic from start. You can read more about nav [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).

> You can read more about figure in [this article](https://www.learningjournal.guru/article/html5/html5-when-to-use-figure-tag/).